contracts/external/TransparentUpgradeableProxy.sol

```diff
@@ -1,2 +1,4 @@
+pragma solidity ^0.8.4;
```

Comment:
Forge will throw a compilation error if a `.sol` file doesn't include the `pragma` declaration. I think fine to add + probably best practice for all solidity files to specify their version

Reply:
Have looked and there isn't currently a way to do this with `forge`. 

Do you have an example of an external file that we are likely to import which doesn't have a `pragma solidity` statement? Surely all external contracts must have it in order for those contracts to be compilable in their home repos?

If this does become a strong need, we can ask the `forge` team about potentially putting it on the roadmap/see if others have the requirement